tools/infer/predict_system.py

```diff
@@ -88,6 +92,15 @@ def __call__(self, img):
             tmp_box = copy.deepcopy(dt_boxes[bno])
             img_crop = self.get_rotate_crop_image(ori_im, tmp_box)
             img_crop_list.append(img_crop)
+            cv2.imwrite(
+                '/home/zhoujun20/dygraph/PaddleOCR_rc/inference_results/{}.jpg'.
+                format(bno), img_crop)
+        if self.use_angle_cls:
+            img_crop_list, angle_list, elapse = self.text_classifier(
+                img_crop_list)
+            print("cls num  : {}, elapse : {}".format(
```

Comment:
print the tme unit at the sam time?

Reply:
Only need to print the whole time is enough